gems/pending/spec/postgres_ha_admin/failover_databases_spec.rb

```diff
@@ -1,132 +1,93 @@
 require 'postgres_ha_admin/failover_databases'
-require 'postgres_ha_admin/postgres_ha_admin'
-require 'spec/support/test_env_helper'
 require 'pg'
 
 describe PostgresHaAdmin::FailoverDatabases do
-  subject do
-    described_class.new(TestEnvHelper::CONFIG_DIR, TestEnvHelper::CONFIG_DIR, :dbname => 'travis', :user => 'travis')
-    # described_class.new(TestEnvHelper::CONFIG_DIR, TestEnvHelper::CONFIG_DIR, :dbname => 'vmdb_test')
+  before do
+    @yml_file = Tempfile.new('failover_databases.yml')
+    @logger_file = Tempfile.new('ha_admin.log')
+    @logger = Logger.new(@logger_file)
+    @failover_databases = described_class.new(@yml_file, @logger)
   end
 
-  # let(:connection) { PG::Connection.open(:dbname => 'vmdb_test') }
-  let(:connection) do
-    begin
-      PG::Connection.open(:dbname => 'travis', :user => 'travis')
-    rescue PG::ConnectionBad => _err
-      skip "travis database does not exist"
-    end
+  after do
+    @yml_file.close
+    @yml_file.unlink
```

Comment:
You can pass `true` to `#close` to simplify this (http://ruby-doc.org/stdlib-2.2.5/libdoc/tempfile/rdoc/Tempfile.html#method-i-close)